t/lib/bash/test_header

```diff
@@ -234,13 +251,13 @@ function commit_sort() {
     # Sort the svn status part of the message
     status_sort $INPUT_FILE $TMP_OUTPUT_FILE
     # Sort the 'Adding/Deleting', etc part of the message
-    python2 -c 'import re, sys
+    python3 -c 'import re, sys
 text = sys.stdin.read()
 sending_lines = re.findall("^\w+ing  +.*$", text, re.M)
 prefix = text[:text.index(sending_lines[0])]
 suffix = text[(text.index(sending_lines[-1]) + len(sending_lines[-1])):]
 sending_lines.sort()
-print prefix + "\n".join(sending_lines) + suffix.rstrip()
+print (prefix + "\n".join(sending_lines) + suffix.rstrip())
```

Comment:
The space between `print` and `(` but I was slightly surprised it was valid. It's unusual.